eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py

```diff
@@ -21,7 +21,7 @@ class Request(db.Model):
   approvals = db.Column(db.Integer)
   created_at = db.Column(db.DateTime)
   updated_at = db.Column(db.DateTime)
-  campaign = db.relationship('Campaign', backref='requests')
+  campaign = db.relationship('Campaign')
```

Comment:
no more backrefs?

Reply:
Yeah. backref wasn't cooperating with the dependent destroy. Give me a sec and I'll test if we can still grab a Campaign through its Request

Reply:
Just confirmed that we can still grab a campaign through its request